prdoc/pr_4755.prdoc

```diff
@@ -0,0 +1,24 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Send PeerViewChange with high priority
+
+doc:
+  - audience: Node Operator
+    description: |
+      - orchestra updated to 0.4.0
```

Comment:
This can be more specific and mention why we upgrading it, in this case it should be the support for susbsytem message delivery prioritization.